src/node.rs

```diff
@@ -93,25 +80,49 @@ impl Branch {
         }
     }
 
-    pub(crate) fn from_filled(children: [NodeId; 8], filled: u8) -> Self {
-        Branch { children, filled }
+    pub fn x0_y0_z0(&self) -> NodeId {
```

Comment:
Does it make sense to mark these 8 small functions with `#[inline]`?

Reply:
Done